cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -1022,11 +1022,12 @@ func WatchConfigMap() {
 	viper.SetConfigType("json")
 	viper.SetConfigName(autoReloadConfigFileName)
 	for {
-		verr := viper.ReadInConfig()
-		if verr == nil {
+		if verr := viper.ReadInConfig(); verr == nil {
 			break
 		} else {
-			glog.V(4).Infof("Can't read the autoreload config file %s/%s due to the error: %v, will retry in 3 seconds", autoReloadConfigFilePath, autoReloadConfigFileName, verr)
```

Comment:
When the file is not found, keep the existing logging at level `4`. What we want is a log at level `1` when the configuration cannot be parsed, such as when there's a missing comma. So, any error that is not `ConfigFileNotFoundError`.

Reply:
Just modified that. Thanks!

Reply:
I think your log statements are reversed. Right?